amd64/container-build.sh

```diff
@@ -0,0 +1,44 @@
+suffix="" # e.g. -hardened
+arch="amd64"
+busybox_version="x86_64"
+dist="http://distfiles.gentoo.org/releases/${arch}/autobuilds/"
+stage3="$(wget -q -O- ${dist}/latest-stage3-${arch}${suffix}.txt | tail -n 1 | cut -f 1 -d ' ')"
+
+mkdir newWorldOrder; cd newWorldOrder
+echo "Downloading and extracting ${stage3}..."
+wget -q -c "${dist}/${stage3}"
+bunzip2 -c $(basename ${stage3}) | tar --exclude "./etc/hosts" --exclude "./sys/*" -xf -
+rm -f $(basename ${stage3})
+wget -q -O /busybox "http://www.busybox.net/downloads/binaries/latest/busybox-${busybox_version}"
```

Comment:
Upstream statically linked busybox, at the moment all coreutils commands and tar, bunzip2, wget works. Hope this will be the same in the future.